### PR TITLE
Create 06-counter-storage.md

### PR DESCRIPTION
**Pull request by josephclander**

### CODE REVIEW

File: pages/v2/07-utility-connectors/03-data-storage/06-counter-storage.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  
+> **_PLEASE NOTE:_** Using Utility Connectors counts toward your overall task usage.
+
+
+</section>
+<section class="card">
+  
+## Purpose
+
+To keep multiple counters, scoped globally or to a Cycle, you can use the **Counter Storage** Connector.
```
Comment (joshuamullings) [inline pages/v2/07-utility-connectors/03-data-storage/06-counter-storage.md:24]:
```suggestion
 The **Counter Storage** Connector allows for multiple counters, either stored globally or within a Cyclr, to be tracked.
```

File: pages/v2/07-utility-connectors/03-data-storage/06-counter-storage.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Counters can be created and updated in one part of the Cycle and retrieved in another.
```
Comment (joshuamullings) [inline pages/v2/07-utility-connectors/03-data-storage/06-counter-storage.md:26]:
```suggestion
Counters can be created and/or updated in one step of a Cycle and retrieved from another step in a Cycle.
```

File: pages/v2/07-utility-connectors/03-data-storage/06-counter-storage.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Cycle Counters are only be accessible _within the same cycle_, where as Global Cycle Counters _persist across your account_.
+
+The **Counter Storage** Connector works by using a Name and an Amount. The Name must be unique. The Amount will be 1 by default and can accept float values.
```
Comment (joshuamullings) [inline pages/v2/07-utility-connectors/03-data-storage/06-counter-storage.md:30]:
```suggestion
The **Counter Storage** Connector stores and/or adjusts an Amount against a unique Counter Name. The Amount will be adjusted by 1 by default and can accept float values.
```

**Review by joshuamullings — CHANGES_REQUESTED** [review]

**Comment by BenCrum** [comment]
Manually added to Confluence.

**Pull request closed without merging**